modules/KalturaSupport/resources/mw.KBaseMediaList.js

```diff
@@ -268,7 +268,7 @@
 		//Media Item
 		renderMediaList: function(callback){
 			//Only render if medialist item are present
-			if (this.getTemplateData().length > 0) {
+			if (this.getTemplateData().length > (0 - !this.getConfig('showEmptyPlaylistError'))) {
 				//Generate new list template data
```

Comment:
We still want the playlist template to be rendered so we can show our UI on the playlist area. For example if you add this to your playlist template we will show a custom UI when there is no loaded playlist: 
<% if (!mediaList || !mediaList.length) { %>
    <div class="ri-playlist-empty-error-wrap"> EMPTY RESULTS</div>
<% } %>